src/main/resources/META-INF/icons/noSubmissions_dark.svg

```diff
@@ -0,0 +1,488 @@
+     id="path901"
+     inkscape:connector-curvature="0" />
+  <path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.00195313"
+     d="m 4.500038,6.043284 c 0,-0.251368 2.27e-4,-0.353909 5e-4,-0.227871 2.75e-4,0.126038 2.75e-4,0.331703 0,0.457031 -2.75e-4,0.125329 -5e-4,0.0222 -5e-4,-0.22916 z"
+     id="path903"
+     inkscape:connector-curvature="0" />
+  <path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.00097656"
+     d="M 3.789117,6.103343 V 5.707347 H 4.1445855 4.500054 V 6.103343 6.499339 H 4.1445855 3.789117 Z"
+     id="path907"
+     inkscape:connector-curvature="0" />
+  <path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.00097656"
+     d="M 3.789117,5.932444 V 5.502268 H 4.1445855 4.500054 V 5.932444 6.36262 H 4.1445855 3.789117 Z"
+     id="path909"
+     inkscape:connector-curvature="0" />
+  <path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.00097656"
+     d="M 3.5000545,6.021311 V 5.543284 h 0.4999995 0.5 v 0.478027 0.478028 h -0.5 -0.4999995 z"
+     id="path911"
+     inkscape:connector-curvature="0" />
+  <path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.00097656"
+     d="m 3.6079645,4.997943 -0.078125,-0.001 -0.00325,-0.0023 c -0.01032,-0.0074 -0.020535,-0.02861 -0.02475,-0.05148 -0.0026,-0.01402 0.00395,-0.110143 0.01193,-0.17521 0.015605,-0.127437 0.04566,-0.247083 0.088063,-0.350586 0.0032,-0.0078 0.00652,-0.01602 0.00739,-0.01831 l 0.0016,-0.0041 H 4.189437 4.7680475 l -0.00445,0.0046 c -0.00823,0.0086 -0.02761,0.02995 -0.041435,0.04567 -0.090668,0.103098 -0.134565,0.175501 -0.1672495,0.275861 -0.01199,0.03681 -0.03136,0.105779 -0.035745,0.12727 -0.00265,0.01286 -0.00934,0.05539 -0.00937,0.0593 -1.1e-5,0.0018 -2.45e-4,0.002 -8.5e-4,0.001 -5.5e-4,-8.5e-4 -7e-4,4.76e-4 -4e-4,0.0034 3.25e-4,0.0035 1.865e-4,0.0044 -5.5e-4,0.0033 -7.5e-4,-0.001 -9e-4,-1.6e-5 -5.5e-4,0.0039 3.5e-4,0.0039 2.18e-4,0.0049 -4.75e-4,0.0034 -6.5e-4,-0.0014 -8e-4,-2.5e-4 -4.25e-4,0.0039 3.75e-4,0.0042 2.375e-4,0.0055 -5e-4,0.0044 -7.5e-4,-0.0011 -8.5e-4,-2.2e-5 -4.25e-4,0.0038 4.75e-4,0.0041 3.5e-4,0.0051 -6.5e-4,0.0045 -9e-4,-5.5e-4 -0.0011,-7.3e-5 -6e-4,0.002 3.25e-4,0.0016 -8.9e-5,0.0081 -9.5e-4,0.01453 -8.5e-4,0.0064 -0.0019,0.01442 -0.00225,0.01772 -4e-4,0.0033 -0.00125,0.0067 -0.00195,0.0075 -0.0038,0.0046 -0.02748,0.01337 -0.04141,0.01535 -0.01845,0.0026 -0.6792295,0.0038 -0.8498505,0.0015 z"
+     id="path919"
+     inkscape:connector-curvature="0" />
+  <path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.00097656"
+     d=""
```

Comment:
Nit: This `path` seems to be empty.

Reply:
How did you find this :smile: did you actually read through the entire svg??

Reply:
We take code reviewing seriously here, don't we? :joy: